src/Sqids/SqidsEncoder.cs

```diff
@@ -12,23 +28,56 @@ public sealed class SqidsEncoder
 	private readonly int _minLength;
 	private readonly string[] _blockList;
 
+#if NET7_0_OR_GREATER
+	/// <summary>
+	/// The minimum numeric value that can be encoded/decoded using <see cref="SqidsEncoder{T}" />.
+	/// This is always zero across all ports of Sqids.
+	/// </summary>
+	public static T MinValue => T.Zero;
+#else
 	/// <summary>
 	/// The minimum numeric value that can be encoded/decoded using <see cref="SqidsEncoder" />.
 	/// This is always zero across all ports of Sqids.
 	/// </summary>
 	public const int MinValue = 0;
+#endif
 
+#if NET7_0_OR_GREATER
+	/// <summary>
+	/// The maximum numeric value that can be encoded/decoded using <see cref="SqidsEncoder{T}" />.
+	/// It's equal to `int.MaxValue`.
+	/// </summary>
+	public static T MaxValue => T.MaxValue;
+#else
 	/// <summary>
 	/// The maximum numeric value that can be encoded/decoded using <see cref="SqidsEncoder" />.
 	/// It's equal to `int.MaxValue`.
 	/// </summary>
 	public const int MaxValue = int.MaxValue;
+#endif
 
+#if NET7_0_OR_GREATER
+	/// <summary>
+	/// Initializes a new instance of <see cref="SqidsEncoder{T}" /> with the default options.
+	/// </summary>
+#else
 	/// <summary>
 	/// Initializes a new instance of <see cref="SqidsEncoder" /> with the default options.
 	/// </summary>
+#endif
 	public SqidsEncoder() : this(new()) { }
 
+
+#if NET7_0_OR_GREATER
+	/// <summary>
+	/// Initializes a new instance of <see cref="SqidsEncoder{T}" /> with custom options.
+	/// </summary>
+	/// <param name="options">
```

Comment:
nit: the `param` section of the xmldoc is the same as the .net6-and-below version; they could be combined.